subprojects/kotlin-dsl-plugins/src/main/kotlin/org/gradle/kotlin/dsl/plugins/dsl/KotlinDslPluginOptions.kt

```diff
@@ -27,7 +29,10 @@ import org.gradle.kotlin.dsl.*
  *
  * @see KotlinDslPlugin
  */
-abstract class KotlinDslPluginOptions {
+abstract class KotlinDslPluginOptions internal constructor(objects: ObjectFactory) {
+
+    private
```

Comment:
I guess formatting visibility in a separate line is intended.

Reply:
Yes